src/actions/modal.js

```diff
@@ -0,0 +1,37 @@
+import {
+  MODAL_OPEN,
+  MODAL_CLOSE,
+  MODAL_SET_CHILDREN,
+  MODAL_LOADING,
+  MODAL_READY,
```

Comment:
If we have MODAL_LOADING, we should use it as toggle param. I think that we don't need the MODAL_READY